<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiClass</class>
 <widget class="QMainWindow" name="guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TRIVIADOR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>611</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>350</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/gui/background.png);</string>
      </property>
      <property name="text">
       <string>PLAY</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>-1</y>
        <width>611</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/gui/startMeniu.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <zorder>frame</zorder>
     <zorder>playButton</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/gui/background.png);</string>
      </property>
      <property name="text">
       <string>LOG IN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signUpButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/gui/background.png);</string>
      </property>
      <property name="text">
       <string>SIGN UP</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>-1</y>
        <width>611</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/gui/startMeniu.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>loginButton</zorder>
     <zorder>signUpButton</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
